### DISCUSSION TITLE
Postgresql connection

### DISCUSSION THREAD

**Discussion by legaljustus** [Budibase/budibase#1492] [Help] [answered]
Hi!

When using the postgresql integration it seems that everytime you preview an app, refresh a deployed app or even move between screens, The integration opens a new sessions on the postgresql server without closing the previous one (it just puts it in idle).

Am I using the connection in the wrong way or is this a (known) issue? 

Thanks for the help,

Justus

**Comment by mjashanks** [comment, marked as answer]
This was resolved by https://github.com/Budibase/budibase/pull/1602
